### DISCUSSION TITLE
Link for arena point generator website?

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#724] [(A) Anything]
What is the link for the website? I need to know because it's not fun to paste all that code into Tryit Editor V3.6. @PatheticMustan

**Comment by hostedposted** [comment]
The website is down.
![](https://user-images.githubusercontent.com/70211235/111200553-47240c00-8598-11eb-991f-c555081b6bb6.JPG)

**Comment by irhrhd** [comment]
What try it editor i want to try

**Reply by hellonearth311** [reply]
[Click here](https://www.w3schools.com/html/tryit.asp?filename=tryhtml_intro) for it.
